plugins/woocommerce-admin/client/marketplace/components/product-card/product-card.tsx

```diff
@@ -38,7 +44,21 @@ function ProductCard( props: ProductCardProps ): JSX.Element {
 	let productVendor: string | JSX.Element | null = product?.vendorName;
 	if ( product?.vendorName && product?.vendorUrl ) {
 		productVendor = (
-			<a href={ product.vendorUrl } rel="noopener noreferrer">
+			<a
+				href={ product.vendorUrl }
+				rel="noopener noreferrer"
+				onClick={ () => {
+					queueRecordEvent(
+						'marketplace_product_card_vendor_clicked',
+						{
+							product: product.title,
+							vendor: product.vendorName,
+							product_type: type,
+							...( product.label && { label: product.label } ),
```

Comment:
I was going to fill this in once I had finalised the product card click below. We should ensure the same data is passed when a vendor name is clicked as when a product link card link is clicked,
```suggestion
							position: product.position,
							...( product.label && { label: product.label } ),
							...( product.group && { group: product.group } ),
							...( product.searchTerm && { search_term: product.searchTerm } ),
							...( product.category && { category: product.category } ),
```